src/CSSBattle/11_EyeOfSauron/WeiPin.html

```diff
@@ -0,0 +1,61 @@
+<div class='rect1'></div>
+<div class='cir1'></div>
+<div class='rect2'></div>
+<div class='cir2'></div>
+<div class='eye'>
+	<div class='iris'>
+  </div>
+</div>
+<style>
+  body {
+    background: #191210;
+    margin:0;
+    display: flex;
+    align-items: center;
+  }
+  .cir1 {
+    position: absolute;
```

Comment:
might want to make position:absolute into a common prop for divs since most of them uses it